src/services/nostr-connect.ts

```diff
@@ -57,6 +57,9 @@ export type NostrConnectErrorResponse = {
   error: string;
 };
 
+// FIXME list all requested perms
+const Perms = "nip04_encrypt,nip04_decrypt,sign_event:0,sign_event:1,sign_event:3,sign_event:4,sign_event:6,sign_event:7"
```

Comment:
Is this supposed to be an extensive list of all events noStrudel could ask the user to sign?

Reply:
It could be the full list, or at least ones which might get signed in the background without explicit user action (30078, etc) or ones that will be signed immediately after login (very annoying to get back to confirmation screen right after it's just been closed).